programming/python/functions.md

```diff
@@ -33,6 +33,6 @@ The same function modified & printing a log message:
 
 ~~~~~ python
 def square(num):
-    print 'found the square of %d' % num
+    print("found the square of {:d}".format(num))
```

Comment:
f-strings?

Reply:
Hrm, didn't think about that. f-strings feel a _bit_ more magic here than `.format`. My goal was to fix things which were either now wrong or are foot-guns. In that light I'm going to leave this as-is, but happy for you to follow-on later and convert these further.